### PR TITLE
gnrc_gomach: add duty-cycle display shell command.

### PR DESCRIPTION
**Pull request by zhuoshuguo**
### Contribution description
Add a shell command to display GoMacH's current, achieved radio duty-cycle.

### Issues/PRs references

See also #8178.

### CODE REVIEW

File: sys/include/net/gnrc/mac/mac.h [hunk 1]
```diff
@@ -56,6 +56,16 @@ extern "C" {
...
 
+
+/**
+ * @brief Enable/disable MAC radio duty-cycle recording and displaying.
+ *
+ * Set "1" to enable, set "0" to disable.
+ */
+#ifndef GNRC_MAC_ENABLE_DUTYCYLE_RECORD
+#define GNRC_MAC_ENABLE_DUTYCYLE_RECORD    (1U)
+#endif
```
Comment (miri64) [inline sys/include/net/gnrc/mac/mac.h]: This looks unrelated. Please move it at least to a separate commit

**Comment by miri64** [comment]
Sorry, I hit enter to quickly. Edited the comment to reflect my later point of the review.

**Comment by zhuoshuguo** [comment]
@miri64 Ping~ Adopted comments and updated~

File: examples/gnrc_networking_mac/mac.c [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2018 SHUGUO ZHUO
```
Comment (miri64) [inline examples/gnrc_networking_mac/mac.c]: Capitalization.

File: examples/gnrc_networking_mac/mac.c [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+int mac_cmd(int argc, char **argv)
+{
+    if (argc < 2) {
+        printf("usage: %s duty\n", argv[0]);
+        return 1;
+    }
+
+    if (strcmp(argv[1], "duty") == 0) {
+
```
Comment (miri64) [inline examples/gnrc_networking_mac/mac.c]: Superfluous newline

File: examples/gnrc_networking_mac/mac.c [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        gnrc_netif_t *netif = NULL;
+        netif = gnrc_netif_iter(netif);
+
+        msg_t msg;
+        msg.type = GNRC_MAC_TYPE_GET_DUTYCYCLE;
+        msg_send(&msg, netif->pid);
+    }
+    else {
+        puts("error: invalid command");
```
Comment (miri64) [inline examples/gnrc_networking_mac/mac.c]: if there is only one command, why not cal l the command `mac_duty` and don't expect an argument?

Comment (zhuoshuguo) [reply examples/gnrc_networking_mac/mac.c]: ... em.. I believe in the future, there will be more MAC commands, and this PR just introduces the first one.

Comment (miri64) [reply examples/gnrc_networking_mac/mac.c]: Ok.

Comment (zhuoshuguo) [reply examples/gnrc_networking_mac/mac.c]: :smiley:

**Comment by zhuoshuguo** [comment]
So, adopted new comments.

**Comment by miri64** [comment]
Please address Travis's issues and squash immediately.

**Comment by miri64** [comment]
(I think the only message you really need to adapt is the one you need to squash anyways ;-).

**Comment by zhuoshuguo** [comment]
Should I keep the three commits you early suggested? Or I just squash them into one commit?

**Comment by miri64** [comment]
No, keep the commits I suggested, but squash the last into 62ef3cf745b9041f45824663bad46be66285f7d3 (which's message also needs shortening I now realize)

**Comment by zhuoshuguo** [comment]
Down and clear!~

**Comment by zhuoshuguo** [comment]
Opps, so no longer than 50 characters. OK, do it again~

**Comment by zhuoshuguo** [comment]
How about now?

**Comment by miri64** [comment]
Now 5f2d084 stays uncompilable.... Wait a moment, I try to fix this...

**Comment by zhuoshuguo** [comment]
Or, maybe we can have just two commits.

**Comment by miri64** [comment]
Then the split-up how you did it still doesn't make any sense ;-) Please wait a minute.

**Comment by zhuoshuguo** [comment]
ok

**Comment by miri64** [comment]
I was careful not to, but please have a look anyways, if I did mess up anything.

**Comment by zhuoshuguo** [comment]
I have checked all 3 new commits, looks OK to me!~ So, let's do it.

**Comment by zhuoshuguo** [comment]
Ping~
